_data/backlog.yml

```
@@ -1,9 +1,135 @@
 cards:
+  - title: Connection Prompting
+    description:
+      - Reintroduce the "Prompt Mode" which asks for your input whether or not to allow a connection. Naturally, make it possible to lock in a decision for the future.
+    projects:
+      - Portmaster
+  - title: Full UI Redesign
+    description:
+      - Completely overhaul the Portmaster user interface to simplify all aspects of it.
+      - Easily block connections from the Monitor, easily block all connection of an app, etc...
+    projects:
+      - Portmaster
+  - title: Network Trust System
+    description:
+      - Remember which networks you trust and what security level should be used within them.
```

There might be some misconception. Currently, saving the security level used in a network is not planned. I'm also unsure about the benefit. Could be discussed, though.
```suggestion
      - Remember which networks you trust and activate additional defenses in untrusted networks, like that Café at the corner.
```